components/automate-chef-io/content/docs/applications.md

```diff
@@ -0,0 +1,162 @@
++++
+title = "Applications Tab"
+description = "Monitor Habitat Services in Chef Automate"
+date = 2019-10-18T18:54:09+00:00
+draft = true
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "applications"
+    weight = 10
++++
+
+<!-- This content is re-worked from the original PR. The original content is used throughout other parts of this PR. -->
+
+## Chef Automate + Chef Habitat
+
+The Chef Automate EAS Applications feature allows you to organize and display your applications data from Chef Habitat in an intuitive way.
+
+### Glossary
+
+_Application_ is the program or group of related programs that you are delivering to users.
+
+Chef Habitat provides automation capabilities for building and delivering applications to any environment.
+Chef Habitat defines the application and its dependencies,configuration, management, and behavior in a `plan.sh` file.
```

Comment:
Might be worth calling out that it is in either a plan.sh file or a plan.ps1 file - to highlight that it works on both Linux and Windows